config/routes.rb

```diff
@@ -1,3 +1,27 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  namespace :api do
```

Comment:
I recommend DRYing up the routes so the name spacing isn't repeated so much.  This is how I started invoices:
  namespace :api do
    namespace :v1 do
      namespace :invoices do
        get '/find', to: 'search#show'
      end
      resources :merchants, only: [:index, :show]
      resources :invoices, only: [:index, :show]
    end
  end
end